golos.referral/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(CMAKE_CXX_FLAGS " -abigen ${CMAKE_CXX_FLAGS} -std=c++17")
```

Comment:
See https://github.com/GolosChain/golos-smart/pull/369. You can use `add_contract_with_abi` in CMakeLists.txt.